src/components/ui/button.tsx

```diff
@@ -10,13 +10,13 @@ const buttonVariants = cva(
     variants: {
       variant: {
         default:
-          'bg-primary text-white hover:bg-primary-light dark:bg-slate-50 dark:text-slate-900 dark:hover:bg-slate-50/90',
+          'bg-primary text-white hover:bg-primary-light dark:bg-slate-50 dark:text-slate-900 dark:hover:bg-slate-50/90 focus-visible:ring-primary-light focus-visible:ring-offset-2',
         destructive:
           'bg-red-500 text-slate-50 hover:bg-red-500/90 dark:bg-red-900 dark:text-slate-50 dark:hover:bg-red-900/90',
         outline:
```

Comment:
🟡 Just an extra thought. Since you're already using variant='outline', which serves a similar visual purpose to variant='filter', maybe we can remove the filter variant definition? It would simplify the code and help avoid any confusion later on, especially since we're planning to reuse these styles. Your thoughts?